dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_acceptInputFileTest/server.xml

```diff
@@ -0,0 +1,22 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server description="Server for testing Faces 4.0 and input file section">
+
+    <include location="../fatTestPorts.xml"/>
+
+    <featureManager>
+        <feature>cdi-4.0</feature>
```

Comment:
I merged https://github.com/OpenLiberty/open-liberty/pull/23750 so CDI is enabled automatically now with faces-4.0 no need to specify in the server.xml.

Reply:
👍🏼 Thanks for the heads up.  I'll keep this the way it is written, but will remove cdi features from future pulls.